react/week1/todolist/src/App.js

```diff
@@ -0,0 +1,64 @@
+import './App.css';
+
+const myTodo = [
+  {
+    id: 1,
+    task: "Finish my hyf homework",
+    dueDate: "Tue 18-05-2021"
+  },
+  {
+    id: 2,
+    task: "Shop groceries",
+    dueDate: "Wed 19-05-2021"
+  },
+  {
+    id: 3,
+    task: "Do laundry",
+    dueDate: "Thu 20-05-2021"
+  }
+]
+
+function App() {
+  return (
+    <div className="App">
+      <h1>Todo List</h1>
+      <div className="container">
+        <Table />
+      </div>
+    </div>
+  );
+}
+
+function Table() {
+  return (
+    <table className="todo-table">
+      <tr>
+        <TableHeader title="Task" />
+        <TableHeader title="Due Date" />
+      </tr>
+      {myTodo.map(list => {
```

Comment:
Think carefuly about naming : 
- myTodo refers to a list of todos
- each element in the array is not a list. Its a todo
- A component named Table is too vague, imo. Call it TodoTable instead